<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>САД</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(234, 234, 234);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>782</width>
        <height>541</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="find_works_tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Главная</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTextEdit" name="search_text">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="searchButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Поиск</string>
           </property>
          </widget>
          <widget class="QPushButton" name="fullSearchButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Расширенный поиск</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>751</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Расширенный поиск</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="2" column="3" colspan="2">
              <widget class="QCheckBox" name="ikrbses_check">
               <property name="text">
                <string>Научный отчёт</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QComboBox" name="priority_sub_combo">
               <item>
                <property name="text">
                 <string>-</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="4" colspan="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Область знаний</string>
               </property>
              </widget>
             </item>
             <item row="0" column="6">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string> Регион</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QComboBox" name="year_combo">
               <item>
                <property name="text">
                 <string>-</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2022</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2021</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2020</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2019</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2018</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2017</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2016</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2015</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2014</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2013</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2012</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2011</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="dissertation_check">
               <property name="text">
                <string>Диссертация</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="6">
              <widget class="QCheckBox" name="rnf_check">
               <property name="text">
                <string>РНФ</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="tristate">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="4" colspan="2">
              <widget class="QComboBox" name="rubrics_combo">
               <item>
                <property name="text">
                 <string>-</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QComboBox" name="priority_combo">
               <item>
                <property name="text">
                 <string>-</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="6">
              <widget class="QComboBox" name="region_combo">
               <item>
                <property name="text">
                 <string>-</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="1" colspan="2">
              <widget class="QCheckBox" name="nioktr_check">
               <property name="text">
                <string>НИОКТР</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="5">
              <widget class="QCheckBox" name="rid_check">
               <property name="text">
                <string>РИД</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Приоритеты</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Области приоритета</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Сортировать за год</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="allWorkTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>751</width>
            <height>351</height>
           </size>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Тип</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Название</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Приоритетное направление</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Исполнитель</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="main_tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Аналитика</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2">
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>ГОД</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">visualButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <layout class="QGridLayout" name="gridLayout"/>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>Месяц</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">visualButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Графическое предстваление</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="download_tab">
       <attribute name="title">
        <string>Загрузка новых работ</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="3" column="0">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="2" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="3">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>РНФ</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>ЕГИСУ НИОКТР</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Источники</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QPushButton" name="downloadButton">
            <property name="text">
             <string>Загрузить</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0" rowspan="2" colspan="6">
           <widget class="QTextBrowser" name="dlOutTextBrowser"/>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Загрузка научных работ</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Оставшееся время загрузки ~ 5 минут</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="visualButtonGroup"/>
 </buttongroups>
</ui>
